### PR TITLE
markdig fixes

### PR DESCRIPTION
**Pull request by mairaw**
An attempt to fix some of the Markdig issues being listed on the report.

### CODE REVIEW

File: xml/System.Windows.Forms/Control.xml [hunk 1]
```diff
@@ -2078,8 +2078,26 @@
   
  For more information about handling events, see [Handling and Raising Events](~/docs/standard/events/index.md).  
   
-   
+## Notes to inheritors
```
Comment (rpetrusha) [inline xml/System.Windows.Forms/Control.xml]: Despite the level-2 head, this renders as level-3.

File: xml/System.Xml/XmlConvert.xml [hunk 2]
```diff
@@ -150,14 +150,14 @@ writer.WriteElementString(XmlConvert.EncodeName("Order Detail"),"My order");
 ## Remarks  
  The names are decoded using the following rules:  
   
--   Names are decoded from left to right.  
+- Names are decoded from left to right.  
   
--   Any sequence _x `HHHH_` (where HHHH stands for a valid, four digit hexadecimal UCS-2 code) that has not been decoded is transformed into the corresponding Unicode 2.1 (Unicode 3.0 if supported by the application) character.  
+- Any sequence \_x `HHHH_` (where HHHH stands for a valid, four digit hexadecimal UCS-2 code) that has not been decoded is transformed into the corresponding Unicode 2.1 (Unicode 3.0 if supported by the application) character.
```
Comment (rpetrusha) [inline xml/System.Xml/XmlConvert.xml]:
```suggestion
- Any sequence \_x`HHHH_` (where HHHH stands for a valid, four-digit hexadecimal UCS-2 code) that has not been decoded is transformed into the corresponding Unicode 2.1 (Unicode 3.0 if supported by the application) character.  
```

File: xml/System.Xml/XmlConvert.xml [hunk 3]
```diff
@@ -220,7 +220,7 @@ writer.WriteElementString(XmlConvert.EncodeName("Order Detail"),"My order");
 ## Remarks  
  This method is similar to the <xref:System.Xml.XmlConvert.EncodeName%2A> method except that it encodes the colon character, which guarantees that the name can be used as the local name part of a namespace qualified name.  
   
- For example, if you passed this method the invalid name a:b, it returns a_x003a_b, which is a valid local name.  
+ For example, if you passed this method the invalid name a:b, it returns a\_x003a\_b, which is a valid local name.  
   
  If `name` is `null` or `String.Empty` then you get the same value returned.
```
Comment (rpetrusha) [inline xml/System.Xml/XmlConvert.xml]:
```suggestion
 If `name` is `null` or <xref:System.String.Empty?displayProperty=fullName>, the method returns the same value.  
```

Comment (rpetrusha) [reply xml/System.Xml/XmlConvert.xml]: A suggestion incidental to your PR.

File: xml/System.Windows.Forms.DataVisualization.Charting/ChartSerializer.xml [hunk 4]
```diff
@@ -432,9 +432,9 @@
   
  This property determines which chart characteristics will not be serialized. Wildcards can be used in the string expression, in the same manner as the <xref:System.Windows.Forms.DataVisualization.Charting.ChartSerializer.SerializableContent%2A> property. For example, to exclude all chart `BackColor` properties from serialization, set this property to "*.BackColor".
```
Comment (mairaw) [inline xml/System.Windows.Forms.DataVisualization.Charting/ChartSerializer.xml]:
```suggestion
 This property determines which chart characteristics will not be serialized. Wildcards can be used in the string expression, in the same manner as the <xref:System.Windows.Forms.DataVisualization.Charting.ChartSerializer.SerializableContent%2A> property. For example, to exclude all chart `BackColor` properties from serialization, set this property to "\*.BackColor".  
```

File: xml/System.Windows.Forms.DataVisualization.Charting/ChartSerializer.xml [hunk 5]
```diff
@@ -432,9 +432,9 @@
...
+ Sometimes, a property can be set to be both serialized and not serialized, which is common when wildcards are used. The <xref:System.Windows.Forms.DataVisualization.Charting.ChartSerializer.NonSerializableContent%2A> expression has a lower priority when compared to the <xref:System.Windows.Forms.DataVisualization.Charting.ChartSerializer.SerializableContent%2A> expression. However, note that less weight is given to string expressions that use wildcards. For example, if the <xref:System.Windows.Forms.DataVisualization.Charting.ChartSerializer.SerializableContent%2A> property is set to "*.BackColor" and the <xref:System.Windows.Forms.DataVisualization.Charting.ChartSerializer.NonSerializableContent%2A> property is set to <xref:System.Windows.Forms.DataVisualization.Charting.ChartArea.BackColor%2A>, all `BackColor` properties except for <xref:System.Windows.Forms.DataVisualization.Charting.ChartArea> objects will be serialized.
```
[the remark this answers is not in the document]
Comment (mairaw) [reply xml/System.Windows.Forms.DataVisualization.Charting/ChartSerializer.xml]: My changes fix that - I noticed that too earlier
image

File: xml/System.Windows.Forms/Control.xml (hunk 1 above)
Comment (mairaw) [reply xml/System.Windows.Forms/Control.xml]: I'm aware. I have an ask to the eng team to consider this section and other notes as H2. Not sure when they will fix though. Ok to keep this change?

**Pull request merged**